### PR TITLE
Raising a PR to merge the contents into main branch

### PR DESCRIPTION
**Pull request by amitmangalvedkar**
Merging the content of `8.10.1-rc` into main branch

### CODE REVIEW

File: azure/bootnode-image/bootnode-image-step-1.sh
Comment (shajeena) [inline azure/bootnode-image/bootnode-image-step-1.sh:1]: sh -i $6 -o StrictHostKeyChecking=no azureuser@$vmip "cd /tmp; curl -skSL 'https://raw.githubusercontent.com/ibm-mas/multicloud-bootstrap/8.10.1-rc/azure/bootnode-image/prepare-bootnode-image.sh' -o prepare-bootnode-image.sh; chmod +x prepare-bootnode-image.sh; sudo su - root -c \"/tmp/prepare-bootnode-image.sh '$ANSIBLE_COLLECTION_VERSION' '$ANSIBLE_COLLECTION_BRANCH' '$BOOTSTRAP_AUTOMATION_TAG_OR_BRANCH'\""

should be changed to 
sh -i $6 -o StrictHostKeyChecking=no azureuser@$vmip "cd /tmp; curl -skSL 'https://raw.githubusercontent.com/ibm-mas/multicloud-bootstrap/main/azure/bootnode-image/prepare-bootnode-image.sh' -o prepare-bootnode-image.sh; chmod +x prepare-bootnode-image.sh; sudo su - root -c \"/tmp/prepare-bootnode-image.sh '$ANSIBLE_COLLECTION_VERSION' '$ANSIBLE_COLLECTION_BRANCH' '$BOOTSTRAP_AUTOMATION_TAG_OR_BRANCH'\""

File: aws/master-cft/byol-existing-ocp/cft-mas-core-dev.yaml
Comment (shajeena) [inline aws/master-cft/byol-existing-ocp/cft-mas-core-dev.yaml:1]: 'git clone -b mas810-alpha https://github.com/ibm-mas/multicloud-bootstrap.git;'

should be changed to 
git clone https://github.com/ibm-mas/multicloud-bootstrap.git;' 

in all CFTs

**Review by NatarajBTI — APPROVED** [review]
Approved this merging of latest code to main branch..

File: aws/master-cft/byol-existing-ocp/cft-mas-core-dev.yaml
Comment (shajeena) [reply aws/master-cft/byol-existing-ocp/cft-mas-core-dev.yaml:1]: we can di these changes after merge . approving !!

**Review by shajeena — APPROVED** [review]

**Pull request merged**